src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,15 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery firstBall = new Lottery();
+        Lottery secondBall = new Lottery();
+        Lottery thirdBall = new Lottery();
+        getLotteryBalls(firstBall,secondBall,thirdBall);
```

Comment:
```suggestion
        Lottery lottery = new Lottery();
        for (int i = 0; i < 3; i++) {
            System.out.println(lottery.getRandomBall());
        }
```
Check naming
when invoking constructor new Lottery(), you are getting an instance of lottery - not ball. That’s why you should call it lottery
Then in for loop get a random ball from 1 lottery. Each time you invoke the method getRandomBall() you will get new random ball